src/discrete-layer/components/system-status/cell-renderer/job-details.header.tsx

```diff
@@ -0,0 +1,161 @@
+import React from 'react';
+import { Box } from '@map-colonies/react-components';
+import './job-details.header.css';
+import { JobModelType, Status } from '../../../models';
+import { IconButton, Tooltip, Typography } from '@map-colonies/react-core';
+import { useIntl } from 'react-intl';
+import CopyToClipboard from 'react-copy-to-clipboard';
+
+interface JobDetailsHeaderProps {
+  job: JobModelType;
+}
+
+// Component should present additional job details:
+// internalId, resourceId, producerName,
+// Fail reason if status failed: reason, if status === Status.Failed
+// Number of tasks specification: failedTasks, inProgressTasks, completedTasks, pendingTasks
+
+export const JobDetailsHeader: React.FC<JobDetailsHeaderProps> = ({
```

Comment:
think worth to use JobModelType instead of distractor syntax + more model compliancy and probably no need ugly in castings